test/models/patient_duplicate_detection_test.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+require 'test_case'
+
+class PatientDuplicateDetectionTest < ActiveSupport::TestCase
+  def setup
+    Patient.destroy_all
+    create(:patient, first_name: 'Deckard', last_name: 'Cain', date_of_birth: '1996-12-31', sex: 'Male', user_defined_id_statelocal: 'asdf123')
+    create(:patient, first_name: 'Deckard', last_name: 'Cain', date_of_birth: '1996-12-31', sex: nil)
```

Comment:
I think we can remove any field throughout this file that defines a value as `nil`